tcmur_cmd_handler.c

```diff
@@ -277,7 +277,8 @@ static int align_and_split_unmap(struct tcmu_device *dev,
 	uint8_t *sense = origcmd->sense_buf;
 	uint64_t opt_unmap_gran;
 	uint64_t unmap_gran_align, mask;
-	int ret, j = 0;
+	int ret = TCMU_NOT_HANDLED;
+	int j = 0;
 	struct unmap_descriptor *desc;
```

Comment:
@lxbsz  I think we need the attached patch along with this. Basically the above case should never happen with the attached patch and the above code is just to handle the compile error.


[unmap-handle-zero.txt](https://github.com/open-iscsi/tcmu-runner/files/1271209/unmap-handle-zero.txt)

Reply:
@mikechristie assuming that was a comment to me, applied your patch. Thanks.

Reply:
Yes, looks good to me.